dial.go

```diff
@@ -0,0 +1,73 @@
+package grpctools
+
+import (
+	"time"
+
+	"github.com/bsm/grpclb"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// Dial creates a client connection.
+func Dial(target string, opts *DialOptions) (*grpc.ClientConn, error) {
+	return DialContext(context.Background(), target, opts)
+}
+
+// DialContext creates a client connection with specified context.
+func DialContext(ctx context.Context, target string, opts *DialOptions) (*grpc.ClientConn, error) {
+	if opts == nil {
+		opts = new(Options)
+	}
+	return grpc.Dial(ctx, target, opts.grpcDialOpts()...)
+}
+
+// --------------------------------------------------------------------
+
+// DialOptions represent dial options.
+type DialOptions struct {
+	// SkipInsecure enables transport security.
+	SkipInsecure bool
+	// SkipBlock makes Dial non-blocking (Dial won't wait for connection to be up before returning).
+	SkipBlock bool
+
+	// LBAddr specifies github.com/bsm/grpclb balancer address, defaults to 127.0.0.1:8383.
+	LBAddr string
+	// SkipLB disables load-balancing.
+	SkipLB bool
```

Comment:
The LB is the only bit where I am not so sure that we need a SkipLB. LBAddr should always be an opt-in not an opt-out.  I wouldn't bother with a default here.